src/app/modules/data-set/dialogs/annotation-consistency-dialog/annotation-consistency-dialog.component.ts

```diff
@@ -83,29 +86,36 @@ export class AnnotationConsistencyDialogComponent implements OnInit {
 
   private getAnnotationConsistencyForAnnotator(): void {
     this.annotationConsistencyService.getAnnotationConsistencyForAnnotator(this.projectId, AnnotationService.getLoggedInAnnotatorId()).subscribe((res: Map<string, string>) => 
-      this.results.set('My consistency', res));
+      this.results.set('Consistency for my annotations', res));
   }
 
   private getAnnotationConsistencyBetweenAnnotatorsForSeverity(): void {
     let severity = this.severityFormControl.value;
     this.annotationConsistencyService.getAnnotationConsistencyBetweenAnnotatorsForSeverity(this.projectId, severity).subscribe((res: Map<string, string>) => 
-      this.results.set('Consistency for severity ' + severity, res));
+      this.results.set('Consistency between annotators for severity ' + severity, res));
   }
 
   private getMetricsSignificanceInAnnotationsForAnnotator(): void {
     this.annotationConsistencyService.getMetricsSignificanceInAnnotationsForAnnotator(this.projectId, AnnotationService.getLoggedInAnnotatorId()).subscribe((res: Map<string, Map<string, string>>) => 
-      this.results.set('My metrics significance', res));
+      this.results.set('Metrics significance for my annotations', res));
   }
 
   private getMetricsSignificanceBetweenAnnotatorsForSeverity(): void {
     let severity = this.severityFormControl.value;
     this.annotationConsistencyService.getMetricsSignificanceBetweenAnnotatorsForSeverity(this.projectId, severity).subscribe((res: Map<string, Map<string, string>>) => 
-      this.results.set('Metrics significance for severity ' + severity, res));
+      this.results.set('Metrics significance between annotators for severity ' + severity, res));
   }
 
   private isValidInput(): boolean {
-    return this.selectedConsistencyType != ''
-      && (this.annotatorNeeded || (this.severityNeeded && this.severityFormControl.valid));
-  }
+    if (this.selectedConsistencyType == '') {
+      this._snackBar.open('Choose consistency type.', 'OK', this.warningSnackBarOptions);
+    } else if (this.severityNeeded && (this.severityFormControl.hasError('min') || this.severityFormControl.hasError('max'))) {
+      this._snackBar.open('Severity must be between 0 and 3.', 'OK', this.warningSnackBarOptions);
```

Comment:
You are hardcoding validation error messages here. Ideally, we would want everything related to the validation of the annotation to be in a single place (e.g., the HTML). I suggest going over https://angular.io/guide/form-validation .